testsuite/features/secondary/minxen_guests.feature

```diff
@@ -41,7 +40,6 @@ Feature: Be able to manage XEN virtual machines via the GUI
 
 @virthost_xen
   Scenario: Parametrize the Xen virtualization host
-    Given I am on the Systems overview page of this "xen_server"
```

Comment:
same question here and in the next scenarios

Reply:
Answered